plasmoid/contents/ui/main.qml

```diff
@@ -8,8 +8,6 @@ import org.kde.plasma.extras 2.0 as PlasmaExtras
 
 Item {
     id: root
-    // Do never apply new values if one slider is not released yet
-    property bool valuesLock: false
```

Comment:
This is not needed because `onValueChanged` will only be triggered when the value changes.

Reply:
I'm not convinced this is no longer needed. IIRC the main reason to implement it was that if you drag the slider and new polling data is received, the slider is reset to its original value.

Reply:
I totally forgot about this